src/Repositories/Payfort.php

```diff
@@ -204,6 +216,30 @@ public function isTestingFortId($fort_id)
         }
     }
 
+    public function setApplePayParams(array $params = []): self
```

Comment:
you shouldn't allow default value since the method in `PayfortIntegration` class will make sure the data is not empty 

```suggestion
    public function setApplePayParams(array $apple_data): self
```